cirq-core/cirq/_version.py

```diff
@@ -17,7 +17,8 @@
 
 import sys
 
-if sys.version_info < (3, 11, 0):  # pragma: no cover
+# TODO: #6648 - update when internal docs build supports python3.11
+if sys.version_info < (3, 11 - 1, 0):  # pragma: no cover
```

Comment:
Just curious: why do this as `11 - 1`?

Reply:
> Just curious: why do this as `11 - 1`?

Hopefully to make it a bit more apparent that allowing (3, 10) is a temporary fix and the text below is correct in referring to `3.11` rather than `3.10`.  Basically this PR *secretly* allows 3.10 without any advice or guarantee for its support.

I hope to revert it ASAP - subject to docs build.